siesta-jackson/src/main/java/org/sonatype/sisu/siesta/jackson/ObjectMapperProvider.java

```diff
@@ -40,8 +39,7 @@ public ObjectMapperProvider() {
         .with(new JacksonAnnotationIntrospector());
     // do not write null values
     mapper.getSerializationConfig()
-        .with(new JacksonAnnotationIntrospector())
-        .withSerializationInclusion(Include.NON_NULL);
```

Comment:
I did some testing and as it stands this writes out null fields in responses, whereas before those were excluded.

To restore the original behaviour you can add the following line:
```
mapper.setSerializationInclusion(Include.NON_NULL);
```